tools/c7n_logexporter/c7n_logexporter/exporter.py

```diff
@@ -189,36 +190,65 @@ def converge_destination_policy(client, config):
                     'Resource': subscription['destination-arn'],
                     'Sid': 'CrossAccountDelivery'}]}))
 
-    def subscribe_account(t_account, subscription):
-        session = get_session(t_account['role'])
+    def subscribe_account(t_account, subscription, region):
+        session = get_session(t_account['role'], region)
         client = session.client('logs')
-
+        sub_name = subscription.get('name', 'FlowLogStream')
+        distribution = subscription.get('distribution', 'ByLogStream')
+        
         for g in account.get('groups'):
-            sub_name = subscription.get('name', 'FlowLogStream')
-            distribution = subscription.get('distribution', 'ByLogStream')
-            filters = client.describe_subscription_filters(
-                logGroupName=g,
-                ).get('subscriptionFilters', ())
-            if filters:
-                f = filters.pop()
-                if (f['filterName'] == sub_name
-                    and f['destinationArn'] == subscription['destination-arn']
-                    and f['roleArn'] == subscription['destination-role']
-                    and f['distribution'] == distribution):
-                    continue
-                client.delete_subscription_filter(
-                    logGroupName=g, filterName=sub_name)
-            client.put_subscription_filter(
-                logGroupName=g,
-                roleArn=subscription['destination-arn'],
-                destinationArn=subscription['destination-arn'],
-                filterName=sub_name,
-                filterPattern="",
-                distribution=distribution)
+            if (g.endswith('*')):
+                g = g.replace('*','')
```

Comment:
this replace would replace all instances of `*` rather than just the last one, could cause issues when matching the log prefix

Reply:
log prefix doesn't support wild cards.